### ISSUE TITLE
Picture tag not working with media breakpoint

### ISSUE THREAD

**Issue by cbaxter713** [alexjoverm/v-lazy-image#36]
Hi @alexjoverm , I'm using v-lazy-image to lazy load some banners that use the picture tag.  I've followed the readme to set this up, and while the picture images do lazy load as expected at mobile dimensions, they are not lazy loading at desktop dimensions above the media breakpoint.

Here is how I am using it:
```html
<v-lazy-image :src="mobileImg"
                  :alt="altText"
                  :intersection-options="{rootMargin: '10%'}"
                  use-picture>
        <source :srcset="desktopImg" :media="breakpointQuery" />
    </v-lazy-image>
```

which is rendering as:
```html
<picture>
    <source srcset="/path/to/mobileImage.jpg" media="(min-width: 800px)">
    <img src="/path/to/desktopImage.jpg" class="v-lazy-image v-lazy-image-loaded" alt="Alt">

</picture>
```

So at screen sizes below 800px, all of the picture tags lazy load the mobile images.  But at screen sizes above 800px, all picture tags are requesting/loading the desktop images upon page load.  Have you seen this issue?

@anthonygore I saw your involvement in getting the picture tag supported.  Have you run across this issue yet? Can you guys reproduce this?

**Comment by anthonygore** [comment]
I'll take a look now

**Comment by anthonygore** [comment]
Okay I think the issue is that the slot contents are not being lazy loaded.

I think the solution is to only render the slot contents when the picture enters the viewport. Will submit a PR to fix it.

Will that cause any bad side effects?

**Comment by cbaxter713** [comment]
I don't believe so, but I guess it depends on how the browser evaluates which src from the <picture> to show, given that the <source> is now injected at a different time. But I think that will work

**Comment by alexjoverm** [comment]
Closed by #37. Thanks to @anthonygore 

Keep in mind this issue might be related to #29 as well, according to the example. Feel free to continue the discussion in there if so.

**Comment by cbaxter713** [comment]
@alexjoverm @anthonygore you guys rock. Thanks for the quick resolution.
